mx.truffleruby/mx_truffleruby.py

```diff
@@ -68,9 +68,14 @@ def ruby_testdownstream(args):
     mx.command_function('build')([])
     mx.run(['ruby', 'tool/jt.rb', 'test', 'fast'])
 
+def ruby_testdownstream_hello(args):
```

Comment:
My gut reaction is this should be called `ruby_testdownstream_basic` or `ruby_testdownstream_spot_check`. But I don't have a huge preference. If you really like `ruby_testdownstream_hello`, that's fine with me.